tests/cmd/stdin-failure2.toml

```diff
@@ -0,0 +1,3 @@
+bin.name = "typos"
+args = "-"
+status.code = 2
```

Comment:
Should we make this case `--write-changes` so we can validate the `2` is for the reason we think?

Reply:
Good idea